include/NAS2D/Mixer/MixerNull.h

```diff
@@ -5,7 +5,7 @@
 namespace NAS2D
 {
 
-	class MixerNull : public Mixer
+	class MixerNull final : public Mixer

```

Comment:
Consider the use of `final` carefully before use. Ref: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-final